### ISSUE TITLE
Missing imports when bundling Zod using canary

### ISSUE THREAD

**Issue by jeremyBanks** [denoland/deno#9097]
I have been working on Deno compatibility for the TypeScript data validation library "Zod" (https://github.com/colinhacks/zod/pull/269). The `v3` development branch is working properly under `deno run`, but I am still encountering some errors in the output of `deno bundle`.

 I am testing with the latest canary build representing the current `master`, 36ff7bd (`deno 1.6.3+36ff7bd (canary, x86_64-unknown-linux-gnu)`).  #9085/https://github.com/swc-project/swc/pull/1318 resolved the error I was previously encountering in #9085, but I am encountering new ones.

when I run `deno bundle https://raw.githubusercontent.com/colinhacks/zod/654680afc2ede388e71e09104eac5a0088fe3207/deno/lib/index.ts && deno run bundle.js`, an error is thrown while evaluating the bundled module:

```
error: Uncaught ReferenceError: util is not defined
const ZodIssueCode = util.arrayToEnum([
                     ^
    at file:///mnt/c/Users/_/src/bundle.js:1:22
```

When we look at the bundle output, we can indeed see that the contents of the `"./helpers/util.ts";` import are not present in the bundled output.

When we don't go through the bundler, but instead just use `deno run https://raw.githubusercontent.com/colinhacks/zod/654680afc2ede388e71e09104eac5a0088fe3207/deno/lib/index.ts` directly, the library is imported and evaluated without any errors.

I experienced the same result (`run` successful, `bundle`-and-`run` missing modules) when I tried bundling a real project of mine that was using Zod and Deno, instead of the above example.

**Comment by kitsonk** [comment]
cc/ @kdy1

**Comment by kdy1** [comment]
Seems like a bug of typescript stripper. I'll fix it soon.

**Comment by kdy1** [comment]
I found that zod uses namespaces.
It's not supported by swc at the moment and I'm not sure implementing it would worth the effort.

**Comment by jeremyBanks** [comment]
@kdy1 Ah, I see, sorry for the confusion. If it's not practical to support, I understand, but perhaps it would be possible to have an error message indicating that they're not supported, so that developers know how to proceed.

**Comment by kdy1** [comment]
@jeremyBanks I already fixed it and published a new version. 😊

**Comment by jeremyBanks** [comment]
Wow, thank you! And thank you in general for all your work on SWC; it's really great for the community to have these great new tools coming along.

**Comment by IllusionPerdu** [comment]
Probably Duplicate of #8478

**Comment by jeremyBanks** [comment]
This fix seems to have been released with Deno 1.7.1. 🙂
